pathfinder/paizo/player_companion/blood_of_the_elements/_blood_of_the_elements.pcc

```diff
@@ -0,0 +1,48 @@
+# CVS $Revision$ $Author$ -- Tue Dec 15 01:48:08 2015 -- reformated by PCGen PrettyLST v6.06.00
+CAMPAIGN:Blood of the Elements
+GAMEMODE:Pathfinder
+RANK:9
+GENRE:Fantasy
+BOOKTYPE:Supplement
+SETTING:Pathfinder|Golarion
+PRECAMPAIGN:4,INCLUDESBOOKTYPE=Core Rules,INCLUDES=Bestiary,INCLUDES=Advanced Player's Guide,INCLUDES=Ultimate Magic
+PUBNAMELONG:Paizo Inc.
+PUBNAMESHORT:Paizo
+PUBNAMEWEB:http://www.paizo.com
+SOURCELONG:Blood of the Elements
+SOURCESHORT:BOTE
+SOURCEWEB:http://paizo.com/products/btpy978p?Pathfinder-Player-Companion-Blood-of-the-Elements
+SOURCEDATE:2014-06
+ISOGL:YES
+
+COVER:blood_of_the_elements
+LOGO:@/publisher_logos/Paizo_Publishing.png
+URL:WEBSITE|http://www.paizo.com/|Visit Paizo Publishing!
+URL:WEBSITE|http://www.paizo.com/communityuse|Paizo's Community Use Policy.
+URL:Paizo Publishing|http://paizo.com/products/btpy978p?Pathfinder-Player-Companion-Blood-of-the-Elements|Buy this publication from Paizo Publishing!
+DESC:Blood of the Elements provides a player-focused, in-depth exploration of the geniekin races and the Elemental Planes.
+
+INFOTEXT:This dataset uses trademarks and/or copyrights owned by Paizo Inc., which are used under Paizo's Community Use Policy. We are expressly prohibited from charging you to use or access this content. This dataset is not published, endorsed, or specifically approved by Paizo Publishing. For more information about Paizo's Community Use Policy, please visit paizo.com/communityuse. For more information about Paizo Publishing and Paizo products, please visit paizo.com.
+COPYRIGHT:Open Game License v 1.0a Copyright 2000, Wizards of the Coast, Inc.
+COPYRIGHT:System Reference Document. Copyright 2000. Wizards of the Coast, Inc.; Authors Jonathan Tweet, Monte Cook, Skip Williams, based on material by E. Gary Gygax and Dave Arneson.
+COPYRIGHT:Pathfinder Player Companion: Blood of the Elements Copyright 2014, Paizo Publishing, LLC; Author: Tim Akers, Judy Bauer, Jim Groves, Chris Lites, Dale C. McCoy, Jr., and Cassidy Werner
+COPYRIGHT:PCGen dataset conversion for "Player Companion: Blood of the Elements" Copyright 2016, PCgen Data team (Including, but not limited to Andrew Wilson (nuance), Paul Grosse, Andrew Maitland)
+
+
+#INFOTEXT:To display an Information in the Source Info section on the Source Materials tab, replace the text after 'INFOTEXT:' with your text and remove the '#'
+
+ABILITY:bote_abilities.lst
+ABILITY:bote_abilities_builder.lst
+ABILITY:abilities_class.lst
```

Comment:
Thanks! I made all the corrections on my end. Sorry there were so many issues. I'm home with a broken foot, so I decided this weekend to learn how to code those books we often use but aren't yet in PCGen. Mostly I've been doing a lot of cut-and-paste and reworking stuff from other books.
I appreciate the help and the advice. Hopefully whatever book I tackle over the next few days has less issues! If there's any you think need worked on, I have time for the next 3 or 4 weeks! LOL
Gwen 

    On Tuesday, April 5, 2016 7:01 PM, Andrew Maitland <notifications@github.com> wrote:
 

 In pathfinder/paizo/player_companion/blood_of_the_elements/_blood_of_the_elements.pcc:> +URL:WEBSITE|http://www.paizo.com/communityuse|Paizo's Community Use Policy.
> +URL:Paizo Publishing|http://paizo.com/products/btpy978p?Pathfinder-Player-Companion-Blood-of-the-Elements|Buy this publication from Paizo Publishing!
> +DESC:Blood of the Elements provides a player-focused, in-depth exploration of the geniekin races and the Elemental Planes.
> +
> +INFOTEXT:This dataset uses trademarks and/or copyrights owned by Paizo Inc., which are used under Paizo's Community Use Policy. We are expressly prohibited from charging you to use or access this content. This dataset is not published, endorsed, or specifically approved by Paizo Publishing. For more information about Paizo's Community Use Policy, please visit paizo.com/communityuse. For more information about Paizo Publishing and Paizo products, please visit paizo.com.
> +COPYRIGHT:Open Game License v 1.0a Copyright 2000, Wizards of the Coast, Inc.
> +COPYRIGHT:System Reference Document. Copyright 2000. Wizards of the Coast, Inc.; Authors Jonathan Tweet, Monte Cook, Skip Williams, based on material by E. Gary Gygax and Dave Arneson.
> +COPYRIGHT:Pathfinder Player Companion: Blood of the Elements Copyright 2014, Paizo Publishing, LLC; Author: Tim Akers, Judy Bauer, Jim Groves, Chris Lites, Dale C. McCoy, Jr., and Cassidy Werner
> +COPYRIGHT:PCGen dataset conversion for "Player Companion: Blood of the Elements" Copyright 2016, PCgen Data team (Including, but not limited to Andrew Wilson (nuance), Paul Grosse, Andrew Maitland)
> +
> +
> +#INFOTEXT:To display an Information in the Source Info section on the Source Materials tab, replace the text after 'INFOTEXT:' with your text and remove the '#'
> +
> +ABILITY:bote_abilities.lst
> +ABILITY:bote_abilities_builder.lst
> +ABILITY:abilities_class.lst
Typographical mistake. Should be "bote_abilities_class.lst", that would be your main concern as to why the bloodlines did not load.—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub